features/step_definitions/your_details_steps.rb

```diff
@@ -43,6 +43,13 @@
   step %{I select my gender as "#{gender}"}
 end
 
+Given(/^my "([^"]*)" "([^"]*)" is below the minimum threshold$/) do |salary, salary_frequency|
```

Comment:
We might not need to create this step, I think there are steps already available that you can use. 
`And my salary is "<salary>" "<salary_frequency>"`
`I enter a salary below the minimum threshold` - 5000 - so assumes `per year`

- I think it is/can be stated in both the Feature and Scenario outline that the salary is below the minimum threshold and so doesn't have to be in the step.

Reply:
I'm not sure I understand this comment as it seems contradictory to the comment you made on this pr, https://github.com/moneyadviceservice/wpcc/pull/142

Reply:
think we've got crossed wires. happy to have a chat about it in person :)